tests/tests/Data/RawCodeCoverageDataTest.php

```diff
@@ -415,14 +411,15 @@ public function testReturnStatementWithOnlyAnArrayWithScalarReturnsTheFirstEleme
 
         $this->assertEquals(
             [
-                8,
+                7,
```

Comment:
In order for our `\SebastianBergmann\CodeCoverage\StaticAnalysis\ExecutableLinesFindingVisitor` to work properly it **MUST** output a _subset_ of the lines discovered by CC Drivers.

Run this script to get the superset (after fixing `Foo::seventh` return type to `string`):

```
cat <<'EOF' | XDEBUG_MODE=coverage php
<?php
xdebug_start_code_coverage();
require "tests/_files/source_with_return_and_array_with_scalars.php";
$a = new Foo();
foreach (get_class_methods($a) as $method) {
    $a->$method();
}
print_r(xdebug_get_code_coverage());
EOF
```

And you'll get:

```
    [tests/_files/source_with_return_and_array_with_scalars.php] => Array
        (
            [8] => 1
            [15] => 1
            [24] => 1
            [30] => 1
            [40] => 1
            [47] => 1
            [54] => 1
            [63] => 1
            [66] => 1
        )
```

In the current state, this PR would produce an empty final CC report over arrays.